atoms/button/index.html

```diff
@@ -0,0 +1,313 @@
+        <span class="btn-svg-f-size">
+          <img class="tui__svg--icon" src="/assets/svgs/heart.svg"  />
+        </span>
+        Icon
+      </button>
+               
+              </code></pre>
+          </div>
+        </li>
+        <li>
+          <h3 class="tui__h-3 tui__m-tb-lg">Circular Edge Buttons</h3>
+          <ul class="tui__bw-sm tui__pd-md tuid__li--component">
+            <li>
+              <button class="tui__btn--primary-br-xl">Primary</button>
+            </li>
+            <li>
+              <button class="tui__btn--link-br-xl">
+                <a href="#">Link</a>
+              </button>
+            </li>
+            <li>
+              <button class="tui__btn--icon-br-xl">
+                <span class="btn-svg-f-size">
+                  <img class="tui__svg--icon" src="/assets/svgs/heart.svg" />
+                </span>
+                Icon
+              </button>
+            </li>
+          </ul>
+          <div class="preformat tui__m-tb-xl">
+            <pre id><code class="language-markup">
+
+    <button class="tui__btn--primary-br-sm">Primary</button>
+
+    <button class="tui__btn--link-br-xl">
+      <a href="#">Link</a>
+    </button>
+
+      <button class="tui__btn--icon-br-xl">
+        <span class="btn-svg-f-size">
+          <img class="tui__svg--icon" src="/assets/svgs/heart.svg"  />
+        </span>
+        Icon
+      </button>
+               
+              </code></pre>
+          </div>
+        </li>
+        <li>
+          <h3 class="tui__h-3 tui__m-tb-lg">Floating action button</h3>
+          <p class="tui__text-base tui__m-sm">
+            Class "tui__btn--float-right"/"tui__btn--float-left" can be used to
+            place button at bottom-right or bottom-left.
+          </p>
+          <ul class="tui__bw-sm tui__pd-md tuid__li--component">
+            <li>
+              <button class="tuid__btn--float-display">Primary</button>
+            </li>
+            <li>
+              <button class="tuid__btn--float-display">
+                <a href="#">Link</a>
+              </button>
+            </li>
+            <li>
+              <button class="tuid__btn--float-display">
+                <span class="btn-svg-f-size">
+                  <img class="tui__svg--icon" src="/assets/svgs/heart.svg" />
+                </span>
+                Icon
+              </button>
+            </li>
+          </ul>
+          <div class="preformat tui__m-tb-xl">
+            <pre id><code class="language-markup">
+
+    <button class="tui__btn--float-right">
+      Primary
+    </button>
+
+    <button class="tui__btn--float-right">
+      <a href="#">Link</a>
+    </button>
+
+    <button class="tui__btn--float-left">
+      <span class="btn-svg-f-size">
+        <img class="tui__svg--icon" src="/assets/svgs/heart.svg" />
+      </span>
+      Icon
+   </button>
+               
+              </code></pre>
+          </div>
+        </li>
+      </ul>
+    </main>
+
```

Comment:
Just a suggestion, follow proper indentation of tags at multiple places in this file, all other files are good